src/sync/Synchroniser.js

```diff
@@ -81,6 +84,39 @@ export class Synchroniser {
     this.batchSize = MIN_SYNC_BATCH_SIZE;
   };
 
+  fetchWrapper = async (url, opts) => {
+    const { body } = opts;
+
+    const bugsnagNotify = (message, response) => {
```

Comment:
Possibly it would be better to have generic bugsnag notifying methods for use all around the app for use outside of this method

Reply:
imo `.notify()` is already quite generic. I don't see any immediate value of making a wrapper for it. wanted to keep things 'dry' with this const method